Makefile

```diff
@@ -7,12 +7,16 @@ install-python-packages:
 
 .PHONY: install-ansible-collections
 install-ansible-collections:
-	ansible-galaxy collection install --upgrade -r tests/integration/requirements.yml
+	ansible-galaxy collection install -r tests/integration/requirements.yml --force
```

Comment:
i think upgrade is preferable to force, so we dont re-install collections with no changes each time this is run